hc/front/tests/test_add_channel.py

```diff
@@ -100,7 +104,24 @@ def test_twiliovoice_works(self):
         alice_channel = User.objects.get(email="alice@example.org")
         alice_before = Channel.objects.filter(user=alice_channel).count()
         self.client.login(username="bob@example.org", password="password")
+        url = "/integrations/add/"
         form = {"kind": "twiliovoice", "value": "+256703357610"}
         self.client.post(reverse("hc-add-channel"), form)
         alice_after = Channel.objects.filter(user=alice_channel).count()
         self.assertEqual(alice_after, (alice_before + 1))
+
+    def test_telegram_works(self):
```

Comment:
This only tests that a telegram integration can be added and not that an actual message is sent. I would have loved to see the test for the actual telegram message